.gitignore

```diff
@@ -15,3 +15,4 @@ debug.log
 docs/output
 docs/includes
 spec/fixtures/evil-files/
+out/
```

Comment:
Nevermind, I suppose I have an `out` folder as well. Maybe that's what the build does now?